manager/src/main/java/org/openremote/manager/map/MapService.java

```diff
@@ -488,9 +491,10 @@ public boolean deleteUploadedFile() {
 
         boolean deleted = false;
         try {
+            connection.close();
```

Comment:
I think `setData()` should do this to cleanup any existing connection before switching